### PR TITLE
Add Latest release and Docker shields to README

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just experimenting with some Jenkins-specific shields based on the YAML metadata provided by the website

jenkins_shields

### Proposed changelog entries

* N/A

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### CODE REVIEW

**Review by jglick — APPROVED** [review]
Neat!

By the way, you can use GitHub’s Markdown preview to see the effect.

**Comment by oleg-nenashev** [comment]
BTW it will not work properly if we merge changelogs before the release
(like we should do?). Please do not merge, will look for a UC-driven
implementation

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 [![][ButlerImage]][website] 
 
 # About
+
+[![Weekly Release](https://img.shields.io/badge/dynamic/yaml.svg?url=https://raw.githubusercontent.com/jenkins-infra/jenkins.io/master/content/_data/changelogs/weekly.yml&label=Weekly%20Release&query=$[-1:].version&color=green)](https://jenkins.io/changelog/)
```
Comment (jvz) [inline README.md]: This is amazing.

Comment (jvz) [reply README.md]: Have you tried probing this endpoint for SSRF exploits? :D

**Review by jvz — APPROVED** [review]
Looks sweet!

**Comment by oleg-nenashev** [comment]
Now it uses the Jenkins update center, so we should not be affected by changelog merges

File: README.md (hunk 1 above)
Comment (oleg-nenashev) [reply README.md]: feel free to do so :P

**Comment by oleg-nenashev** [comment]
retriggering CI

**Comment by oleg-nenashev** [comment]
Will merge it tomorrow if no negative feedback

**Review by timja — APPROVED** [review]

**Pull request merged**